README.md

```diff
@@ -175,7 +175,7 @@ Any aberrant results should be investigated rectified within the data set before
 
 anonlink will garble personally identifiable information (PII) in a way that it can be used for linkage later on. The CODI PPRL process garbles information a number of different ways. The `garble.py` script will manage executing anonlink multiple times and package the information for transmission to the linkage agent.
 
-`garble.py` requires 3 different inputs:
+`garble.py` requires 2 different inputs:
 1. The location of a CSV file containing the PII to garble
 1. The location of a directory of anonlink linkage schema files
 1. The location of a secret file to use in the garbling process - this should be a text file containing a single hexadecimal string of at least 128 bits (32 characters); the `testing-and-tuning/generate_secret.py` script will create this for you if require it, e.g.:
```

Comment:
I think we can make this whole thing simpler by merging line 187 below into this line, something like:
```suggestion
`garble.py` accepts the following positional inputs:
1. (optional) The location of a CSV file containing the PII to garble. If not provided, the script will look for the newest `pii-TIMESTAMP.csv` file in the `temp-data` directory.
1. (required) The location of a directory of anonlink linkage schema files
1. (required)  The location of a secret file to use in the garbling process - this should be a text file containing a single hexadecimal string of at least 128 bits (32 characters); the `testing-and-tuning/generate_secret.py` script will create this for you if require it, e.g.:
```
Then we can remove 187 altogether

I feel like it's a little unusual to have a positional argument in the middle be optional, but as long as it works it's fine. We should make sure to test that thoroughly in case there's something that can confuse the arg parser.

Reply:
Suggestions looks good to me. I think this is robust because I gave the argument the `nargs=?` parameter but am not beyond a reasonable doubt on that.